modules/digitaluniverse/rendering/renderableplanescloud.cpp

```diff
@@ -524,15 +524,17 @@ void RenderablePlanesCloud::initializeGL() {
 }
 
 
-void RenderablePlanesCloud::deleteDataGPU() {
-    for (RenderingPlane& renderingPlane : _renderingPlanesArray) {
-        glDeleteVertexArrays(1, &renderingPlane.vao);
-        glDeleteBuffers(1, &renderingPlane.vbo);
+void RenderablePlanesCloud::deleteDataGPUAndCPU() {
+    for (std::pair<int, PlaneAggregate> pAMapItem : _planesMap) {
+        glDeleteBuffers(1, &pAMapItem.second.vbo);
+        glDeleteVertexArrays(1, &pAMapItem.second.vao);
+        pAMapItem.second.planesCoordinates.clear();
     }
+    _planesMap.clear();
```

Comment:
The bug didn't show up because this clear will also clear the PlaneAggregates

Reply:
Shame on me... :-)